manual/query_builder/idempotence/README.md

```diff
@@ -116,6 +116,45 @@ Statement statement =
 assert !statement.isIdempotent();
 ```
 
+The generic `remove` method is however safe since collection removals are idempotent:
+
+```java
+Statement statement =
+    update("foo")
+        .remove("l", literal(Arrays.asList(1, 2, 3)))
+        .whereColumn("k").isEqualTo(bindMarker())
+        .build();
+// UPDATE foo SET l=l-[1,2,3] WHERE k=?
+assert !statement.isIdempotent();
```

Comment:
shouldn't the `statement.isIdempotent()` return true? - according to your docs:
>The generic `remove` method is however safe since collection removals are idempotent

Reply:
Correct, thanks for pointing this out.